tests/__init__.py

```diff
@@ -0,0 +1,17 @@
+import os
+import sys
+import unittest
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+
+
+def load_tests(loader, tests, pattern):
+    """Test loader function for unittest discovery."""
+    test_suite = unittest.TestSuite()
+    for all_test_suite in unittest.defaultTestLoader.discover(os.path.dirname(__file__), pattern='test_*.py'):
+        for test_suite in all_test_suite:
+            test_suite.addTests(test_suite)
+    return test_suite
+
+if __name__ == '__main__':
+    unittest.TextTestRunner(verbosity=2).run(load_tests())
```

Comment:
Normally I'd encourage adding tests but can we actually hold off on this one

Reason being, I'd like to build out a proper base using pytest + tox rather than unittest for better/native support in packaging pipelines and such. There's also a few specifics I'd like to make sure get implemented properly within pytest, since certain types of tests related to an internet-reliant project like this are wholly incompatible with packaging

Plans are similar to the changes I made in https://github.com/sherlock-project/sherlock/pull/2127 and similar to the mock data found in the test suite for https://github.com/FrostyX/fedora-review-service (the former uses unitttest of course, but referring to the mocks)